### ISSUE TITLE
Wait timeout reports 0ms when timeout with default maxwait

### ISSUE THREAD

**Issue by mgurov** [rhuss/docker-maven-plugin#274]
Given wait configuration:

````xml
                <wait>
                  <log>never to be seen</log>
                </wait>
````

the output of the `docker:run` after 10 seconds is:
````
[ERROR] DOCKER> [local/bes] : Timeout after 0 ms while waiting on log out 'never to be seen'
````

Expected the actual time waited or maxwait expected to be reported.

**Comment by rhuss** [comment]
Thanks for the report. Indeed if the default timeout occurs it should be reported. 'will apply the PR ASAP.  In the meantime, could you please sign-off you commit as described [here](https://github.com/rhuss/docker-maven-plugin/blob/master/doc/contributing.md)

Thanks !

**Comment by mgurov** [comment]
Signed off. I did have a feeling that I missed something.

**Comment by rhuss** [comment]
Applied PR to branch integration. Available in 0.13.4.

Thanks !
